src/cityreader/cityreader.py

```diff
@@ -33,24 +33,16 @@ def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   with open('cities.csv', mode='r') as citiescsv:
       read = csv.DictReader(citiescsv, delimiter=',')
```

Comment:
Awesome job with using DictReader, it makes your `for... in` more semantic and easier to read when passing the args into City()